src/main/java/com/marek/onlinebookstore/controller/OrderController.java

```diff
@@ -44,27 +44,26 @@ public List<OrderDto> getOrderHistory(Authentication authentication, Pageable pa
     }
 
     @GetMapping("/{orderId}/items")
-    @PreAuthorize("hasAnyRole('ROLE_ADMIN', 'ROLE_USER')")
+    @PreAuthorize("hasRole('ROLE_USER')")
     @Operation(summary = "get all items",
             description = "get all items from your order")
     public List<OrderItemDto> getItemsFromOrder(@PathVariable Long orderId) {
         return orderService.getFromOrder(orderId);
     }
 
     @GetMapping("/{orderId}/items/{id}")
-    @PreAuthorize("hasAnyRole('ROLE_ADMIN', 'ROLE_USER')")
+    @PreAuthorize("hasRole('ROLE_USER')")
     @Operation(summary = "get one item",
             description = "get one item from your order")
     public OrderItemDto getOneItemFromOrder(@PathVariable Long orderId, @PathVariable Long id) {
         return orderService.getFromOrder(orderId, id);
     }
 
-    @PutMapping("/{id}")
-    @PreAuthorize("hasAnyRole('ROLE_ADMIN')")
+    @PatchMapping("/{id}/status")
```

Comment:
According to the task description there should be endpoint `PATCH: /api/orders/{id}`